media/walkthroughs/discoverAdt/adt-create.md

```diff
@@ -0,0 +1,29 @@
+<!-- markdownlint-disable MD033 MD041 MD013-->
+
+<figure align="center">
+  <img src="create-ansible-creator.gif" alt="image missing" />
+  <figcaption>Scaffolding a collection using Ansible Creator inside the Ansible extension</figcaption>
```

Comment:
We should check with @tima, but I think all the products need so be Ansible something...Captial Ansible lowercase something.

e.g. Ansible creator

Reply:
All changed to lowercase (except Molecule).